package.json

```diff
@@ -40,10 +39,12 @@
     "lib/transport-utils.js"
   ],
   "devDependencies": {
-    "seneca": "plugin",
+    "async": "1.4.2",
+    "bench": "^0.3.5",
     "docco": "0.7.0",
     "jshint": "2.8.0",
     "mocha": "2.1.0",
+    "seneca": "plugin",
     "seneca-transport-test": "0.1.3"
```

Comment:
@marcopiraccini Is doing some work on seneca-transport-test. Can we if he is finished and update?